stac_fastapi/pgstac/stac_fastapi/pgstac/models/links.py

```diff
@@ -149,14 +149,14 @@ def link_prev(self) -> Optional[Dict[str, Any]]:
                 href = merge_params(self.url, {"token": f"prev:{self.prev}"})
                 return dict(
                     rel=Relations.previous.value,
-                    type=MimeTypes.json.value,
+                    type=MimeTypes.geojson.value,
                     method=method,
                     href=href,
                 )
             if method == "POST":
                 return {
                     "rel": Relations.previous,
-                    "type": MimeTypes.json,
+                    "type": MimeTypes.geojson,
```

Comment:
I'm not sure I see why we use `.value` in one case but not the other. Obviously you didn't introduce this asymmetry but I wonder if you got some sense of what the distinction is while working on this portion of the library

Reply:
I'd wondered about that too, but didn't go down that rabbit hole. It looks like with the way MimeTypes is defined (`MimeTypes(str, Enum)`) that these are identical. We should remove the use of `value` from all the MimeTypes.